### PR TITLE
IMPROVE: provide `nonce` and `IdToken RawString` in Login Result

### PR DESCRIPTION
**Pull request by LiYing2010**
Change List:
1. add `nonce` to `LineLoginResult`
2. add `rawString` to `LineIdToken`
3. source code refactor for `LineLoginResult`

### CODE REVIEW

File: line-sdk/src/main/java/com/linecorp/linesdk/auth/LineLoginResult.java [hunk 1]
```diff
@@ -147,6 +165,16 @@ public LineProfile getLineProfile() {
         return lineProfile;
     }
 
+    /**
+     * Gets the ID token raw string.
+     *
+     * @return A ID token raw string.
+     */
+    @Nullable
+    public String getLineIdTokenRawStr() {
```
Comment (plateaukao) [inline line-sdk/src/main/java/com/linecorp/linesdk/auth/LineLoginResult.java]: please use full name `getLineIdTokenRawString()`

File: line-sdk/src/main/java/com/linecorp/linesdk/internal/IssueAccessTokenResult.java [hunk 2]
```diff
@@ -39,27 +44,36 @@ public InternalAccessToken getAccessToken() {
         return scopes;
     }
 
+    @Nullable
+    public String getIdTokenRawStr() {
```
Comment (plateaukao) [inline line-sdk/src/main/java/com/linecorp/linesdk/internal/IssueAccessTokenResult.java]: use full name `getIdTokenRawString()`

File: line-sdk/src/main/java/com/linecorp/linesdk/internal/IssueAccessTokenResult.java [hunk 3]
```diff
@@ -20,12 +20,17 @@
...
 
-    public IssueAccessTokenResult(
-            @NonNull InternalAccessToken accessToken, @NonNull List<Scope> scopes, @Nullable LineIdToken idToken) {
+    public IssueAccessTokenResult(@NonNull InternalAccessToken accessToken,
+                                  @NonNull List<Scope> scopes,
+                                  @Nullable final String idTokenRawStr,
+                                  @Nullable LineIdToken idToken) {
         this.accessToken = accessToken;
         this.scopes = Collections.unmodifiableList(scopes);
+        this.idTokenRawStr = idTokenRawStr;
```
Comment (plateaukao) [inline line-sdk/src/main/java/com/linecorp/linesdk/internal/IssueAccessTokenResult.java]: idTokenRawString is the raw string of LineIdToken, why not put it as a member of LineIdToken?

Comment (LiYing2010) [reply line-sdk/src/main/java/com/linecorp/linesdk/internal/IssueAccessTokenResult.java]: I thought about this option at first, but I was not sure if it's a good idea to put `raw string` and `parsed fields` together

I will fix it now, if you think it's better

File: line-sdk/src/main/java/com/linecorp/linesdk/auth/LineLoginResult.java (hunk 1 above)
Comment (LiYing2010) [reply line-sdk/src/main/java/com/linecorp/linesdk/auth/LineLoginResult.java]: removed this

File: line-sdk/src/main/java/com/linecorp/linesdk/internal/IssueAccessTokenResult.java (hunk 2 above)
Comment (LiYing2010) [reply line-sdk/src/main/java/com/linecorp/linesdk/internal/IssueAccessTokenResult.java]: removed this

File: line-sdk/src/main/java/com/linecorp/linesdk/internal/IssueAccessTokenResult.java (hunk 3 above)
Comment (LiYing2010) [reply line-sdk/src/main/java/com/linecorp/linesdk/internal/IssueAccessTokenResult.java]: fixed

**Pull request merged**